oleesir/authors-haven
# Create signup page and verify user

**What does this Merge request do?**

Implements feat__createSignup

**Description of Tasks completed?**

Implements feat__createSignup

**How should this be manually tested?**

- git fetch on your local machine and check out to feat__createSignup branch
- Run "yarn start" to start the development server

**Any background context you want to provide?**

N/A


**Screenshots (if appropriate)**

<img width="1422" alt="Screenshot 2021-11-20 at 2 12 58 PM" src="https://user-images.githubusercontent.com/17679511/142727740-ac107511-47f1-44e7-8622-e33a6044b6c1.png">
<img width="1422" alt="Screenshot 2021-11-24 at 1 18 06 PM" src="https://user-images.githubusercontent.com/17679511/143237432-33a94689-2644-497d-81cc-dab1bd17890a.png">
